builder/Dockerfile

```diff
@@ -0,0 +1,66 @@
+# Default output dir containing build artifacts
```

Comment:
Is this the same as the `builder` step on the main `Dockerfile`?

If so, we should use the main `Dockerfile` and build using `--target` option. That way we don't need to keep track of separated files and versions

Also, did you build this manually and upload it right? We may need to automate this since once someone changes the base versions, the builder for that release will need to be updated as well.

Reply:
i agree on both, i was focused on making it work, my next step will be automating the docker image builds.